File: wrappers/csharp/Intel.RealSense/Types/Stream.cs
using System;
using System.Collections.Generic;
using System.Runtime.InteropServices;

namespace Intel.RealSense
{
    public enum Stream
    {
        Any = 0,
        Depth = 1,
        Color = 2,
        Infrared = 3,
        Fisheye = 4,
        Gyro = 5,
        Accel = 6,
        Gpio = 7,
        Pose = 8,
        Confidence = 9,
    }
}

File: wrappers/matlab/stream.m
classdef stream < int64
    enumeration
        any_stream  ( 0)
        depth       ( 1)
        color       ( 2)
        infrared    ( 3)
        fisheye     ( 4)
        gyro        ( 5)
        accel       ( 6)
        gpio        ( 7)
        pose        ( 8)
        confidence  ( 9)
        count       (10)
    end
end